starwars/src/components/PeopleGrid.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from "react";
+// import { people } from "../data";
+import PeopleCard from "./PeopleCard";
+import styled from "styled-components";
+import axios from "axios";
+
+const CardContainer = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+`;
+
+const PeopleGrid = () => {
+    const [peopleState, setPeopleState] = useState([]);
+    console.log(peopleState)
+
+    useEffect(() => {
+        axios
+        .get('https://swapi.co/api/people/')
+        .then(response => {
+            const PeopleArr = response.data.results;
+            // console.log("Testing", peeps);
+            setPeopleState(PeopleArr)
+        })
+    }, []);
+
+    return (
+        <CardContainer className="card-container">
+            {peopleState.map((key) =>(
+                <PeopleCard 
```

Comment:
remember there is the individual item being passed AND the index. so you would need `.map(person, key)` and then key would be key={key} and your other people would be name={person.name}